config.go

```diff
@@ -219,6 +220,24 @@ func (h hypervisor) defaultBridges() uint32 {
 	return h.DefaultBridges
 }
 
+func (h hypervisor) internetworkingModel() (vc.NetInterworkingModel, error) {
```

Comment:
Could you please rename every `internetworkingModel` occurrences into `interNetworkingModel`.